experimental/PropertyDDS/packages/property-changeset/src/changeset.ts

```diff
@@ -618,6 +618,7 @@ export class ChangeSet {
 
 			// Check, whether we have a collision in a path update
 			for (let j = 0; j < paths.length; j++) {
+				io_rebasePropertyChangeSet[typeid] = io_rebasePropertyChangeSet[typeid] ?? {};
```

Comment:
For me this line looks very confusing: I started to ask myself if there is any special logic around this object? Eventually, why can't we just have a simple additional check for `io_rebasePropertyChangeSet[typeid]` in the `if` statement one line below?

Reply:
It would be equivalent logic. I just added similar logic as in this [line](https://github.com/microsoft/FluidFramework/pull/14365/files#diff-92fd4f227a8c38ccab72d6f9673d515691a2553742c7b64cf2c2a6c034090518L622)

Reply:
I didn't mean that it's different, only that it's a bit confusing.

Reply:
I did use a similar pattern in other places of the code. It is a rather common way to default initialize a variable.

In this case, there would be a difference between @sharptrip suggestion and the current code. The current code will have the side effect to create this property, but then the if chek will fail (because the path does not exist). @nedalhy Is this side effect intended?

Reply:
It wasn't my intention to have this side-effect.